aws-rhel8-quay.json

```diff
@@ -208,11 +186,13 @@
         "echo '** Shreding sensitive data ...'",
         "dnf clean all",
         "rm -rf /var/cache/yum /var/cache/dnf",
-        "shred -u /etc/ssh/*_key /etc/ssh/*_key.pub",
-        "shred -u /root/.*history /home/*/.*history",
-        "shred -u /root/.ssh/authorized_keys /home/*/.ssh/authorized_keys",
-        "shred -u /tmp/pull-secret.txt",
-        "shred -u /var/run/user/1000/containers/auth.json",
+        "shred -uf /tmp/key/rh-cdn.pem",
```

Comment:
Will shred leave the empty /tmp/key directory? Should a rmdir be added?

Reply:
directories like that in /tmp will be cleared out by the system, generally on the next reboot.